src/stylesheets/index.scss

```diff
@@ -0,0 +1,2 @@
+@import 'variables.scss';
```

Comment:
We should use @use instead of @import. @import is being phased out by SASS.

Reply:
@use only imports it into the current file

Reply:
We should be fine to use @import for this project, as it's not being deprecated for another year and a half